daemon/src/db.rs

```diff
@@ -508,15 +531,40 @@ async fn load_cfd_row(conn: &mut Transaction<'_, Sqlite>, id: OrderId) -> Result
     .await?
     .ok_or(Error::OpenCfdNotFound)?;
 
+    let role = cfd_row.role;
+    let counterparty_network_identity = cfd_row.counterparty_network_identity;
+
+    // This is in place temporarily as long as the legacy and libp2p connection are used in parallel
+    // The peer-id is defaulted to a placeholder dummy in the database. Based on the dummy we can
+    // decide to default the peer-id according to known makers.
+    let counterparty_libp2p_identity =
+        if cfd_row.counterparty_libp2p_identity == Libp2pIdentity::placeholder() {
+            match role {
+                // Default known maker PeerIds based on the legacy network identity
+                Role::Taker => (match counterparty_network_identity.to_string().as_str() {
+                    MAINNET_MAKER_ID => Some(MAINNET_MAKER_PEER_ID),
+                    TESTNET_MAKER_ID => Some(TESTNET_MAKER_PEER_ID),
+                    DEV_MAKER_ID => Some(DEV_MAKER_PEER_ID),
```

Comment:
🤔 We have the network available every time we call `load_cfd_row`. We could pass it in and use it here instead of these constants.

Reply:
Incorrect, we don't have that information when we archive closed/failed CFDs :(

Reply:
yep. I really did not like duplicating this, but bare in mind that all of this is temporary. Once the legacy code is removed all of this will go away. I think for rolling libp2p out ("giving rollover a try") it is acceptable to do this.